lte/gateway/c/oai/include/amf_app_messages_def.h

```diff
@@ -0,0 +1,60 @@
+/**
+ *  * Copyright 2020 The Magma Authors.
+ *  *
+ *  * This source code is licensed under the BSD-style license found in the
+ *  * LICENSE file in the root directory of this source tree.
+ *  *
+ *  * Unless required by applicable law or agreed to in writing, software
+ *  * distributed under the License is distributed on an "AS IS" BASIS,
+ *  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  * See the License for the specific language governing permissions and
+ *  * limitations under the License.
+ * */
+/*****************************************************************************
+ *
+ *  Source      amf_app_messages_def.h
+ *
+ *  Version     0.1
+ *
+ *  Date        2020/09/08
+ *
+ *
+ * Subsystem   Access and Mobility Management Function
+ *
+ * Description Defines Access and Mobility Management Messages
+ *
+ *******************************************************************************/
+* /
+    // WARNING: Do not include this header directly. Use intertask_interface.h
```

Comment:
is this comment necessary? can this be weeded out?

Reply:
I think this is required as in beginning of our development, we we including this file directly into other files, that was giving many errors, so for future developments it will serve as guidance for developers.